src/main/resources/application.properties

```diff
@@ -2,3 +2,5 @@ spring.application.name=booking-java-bootcamp
 
 springdoc.swagger-ui.enabled=true
 springdoc.swagger-ui.path=/swagger.html
+// this is very important configuration, don't remove it! This missing line caused me apprx. 4 hour of pain :))
```

Comment:
For comments, it is usually best to add a comment in which to describe what this fixes/addresses. As is, when looking over this (regardless of who & when - since you might come back to this after some time and not recall the reasoning for it), it will be hard to understand what it is needed for.

Reply:
:)). I just added this comment because I wanted to express my pain when I discovered that why the POST call isn't working on swagger. :))

Reply:
I know the comment is not that helpful but to me that time was so funny :)).

Reply:
Can we leave it ? Its harmless :)...

Reply:
Fixed.